mod.config

```diff
@@ -0,0 +1,8 @@
+# The id of the mod to launch by default.  This must exist as a directory in the mods directory
+MOD_ID="ra2"
+
+# Import the default OpenRA mods for use as external mod references
+# in your mod.yaml. Accepts values "True" or "False".
+# WARNING: This setting is provided to simplify early mod development
+# and must be disabled before installers can be generated
+INCLUDE_DEFAULT_MODS="True"

```

Comment:
You should aim to make whatever fixes you need to reset this to "False" in the shortish term.